x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -391,8 +378,9 @@ export const EndpointList = () => {
         field: 'metadata.agent.version',
         width: '9%',
         name: i18n.translate('xpack.securitySolution.endpoint.list.endpointVersion', {
-          defaultMessage: 'Version',
+          defaultMessage: 'Sensor',
```

Comment:
we should revert this back to "Version".  The mock as also been corrected in the original ticket

Reply:
gotcha